### DISCUSSION TITLE
growth data 1: same species, unit: OD, same media, different plate readers

### DISCUSSION THREAD

**Discussion by hariszaf** [msysbio/bacterial_growth#24] [Q&A]
What issues we need to consider.

What is comparable - what is not. 

What to implement.

**Comment by hallucigenia-sparsa** [comment]
Indeed, OD can vary from plate reader to plate reader for the same species, medium, and experimental set-up (pre-culture, well plate, wavelength etc.). So yes, if we want to plot ODs from experiments performed with different plate readers, we may have to normalize them. Since we can't easily match plate reader names in the experiment description, we can simplify things by assuming that all biological replicates in the same experiment were performed with the same plate reader (if a plate reader was involved) and to normalize by default across different experiments when visualizing ODs. However, I recommend to give users control about whether to normalize or not, since sometimes the same plate reader (or different plate readers with comparable ODs) may have been used in different experiments, especially if all experiments belong to the same study.

**Comment by hariszaf** [comment]
@ccvdvelde this is just for you to get a notification.
